### PR TITLE
Event wrappers

### PR DESCRIPTION
**Pull request by jakemac53**
fixes https://github.com/dart-lang/polymer-dart/issues/598

### CODE REVIEW

**Comment by jakemac53** [comment]
whoops wrong merge target, closing

File: lib/polymer_interop.dart [hunk 1]
```diff
@@ -1,102 +1,303 @@
...
+  /// The list also includes e.g. text nodes and comments. To skip them, use
+  /// `this.children` instead.
+  List<Node> get childNodes => _proxy['childNodes'];
+
+  /// Read-only property that returns a live `List` of the child elements of
+  /// `this`. The list doesn't include text nodes and comments.
+  List<Element> get children => _proxy['children'];
+
+  // TODO: add docs
+  PolymerClassList get classList => new PolymerClassList(this);
```
Comment (zoechi) [inline lib/polymer_interop.dart]: shouldn't this be `classes` like its named in dart:htmls `Element`?

File: lib/polymer_interop.dart [hunk 2]
```diff
@@ -1,102 +1,303 @@
...
+  Element get firstElementChild => _proxy['firstElementChild'];
 
-  static void endOfMicrotask(f()) {
-    if (!checkExists()) return null;
-    _polymer.callMethod('endOfMicrotask', [() => f()]);
+  /// Sometimes it’s necessary to access the elements which have been distributed
+  /// to a given <content> insertion point or to know to which <content> a given
+  /// node has been distributed. The getDistributedNodes and
+  /// getDestinationInsertionPoints methods, respectively, provide this information.
+  List<Node> getDestinationInsertionPoints() =>
```
Comment (zoechi) [inline lib/polymer_interop.dart]: Would be great to find here whats the difference between `getDestinationInsertionPoints()` and `getDistributedNodes()`
From the names I would expect that the former returns the `<content>` elements and the later the nodes distributed to a specific `<content>` element but I would expect to be able to pass the `<content>` element I want the nodes from. Quite confusing.

File: lib/polymer_interop.dart (hunk 1 above)
Comment (jakemac53) [inline lib/polymer_interop.dart]: Good point, it probably should

File: lib/polymer_interop.dart (hunk 2 above)
Comment (jakemac53) [inline lib/polymer_interop.dart]: https://www.polymer-project.org/1.0/docs/devguide/local-dom.html talks a bit more about these, but it doesn't specify the behavior of getDistributedNodes if you have more than one `<content>` element.

Comment (zoechi) [inline lib/polymer_interop.dart]: Ok, I forgot the instance is already initialized with a specific `content` element previously.

**Pull request closed without merging**